src/components/general/mediaTypes/ImagePopup.tsx

```diff
@@ -0,0 +1,174 @@
+                                    enableReinitialize={true}
+                                    validate={ValidateAnnotation}
+                                    onSubmit={async (form) => {
+                                        await new Promise((resolve) => setTimeout(resolve, 500));
+
+                                        /* Submit new Annotation */
+                                        if (!selectedAnnotation.body.length) {
+                                            /* Patch Annotation */
+                                            SubmitAnnotation([form.annotationValue], 'insert');
+                                        } else {
+                                            /* Insert Annotation */
+                                            SubmitAnnotation([form.annotationValue], 'patch')
+                                        }
+                                    }}
+                                >
+                                    {({ errors }) => (
+                                        <Form>
+                                            <Row>
+                                                <Col>
+                                                    <Field name="annotationValue"
+                                                        type="text"
+                                                        innerRef={tooltipFieldRef}
+                                                        className="w-100 formField fs-4 py-1"
+                                                    />
+
+                                                    {errors.annotationValue &&
+                                                        <p className="fs-5 c-denied mt-2">
+                                                            {errors.annotationValue as string}
+                                                        </p>
+                                                    }
+                                                </Col>
+                                                <Col className="col-md-auto ps-0">
+                                                    <button type="submit"
+                                                        className="secondaryButton fs-4 px-3"
+                                                    >
+                                                        Save
+                                                    </button>
+                                                </Col>
+                                            </Row>
+                                        </Form>
+                                    )}
+                                </Formik>
+                            </div>
+
+                            <div className={classInfoBlock}>
+                                <Row>
+                                    <Col>
+                                        <p className="fs-4">
+                                            {(selectedAnnotation?.body.length) ?
```

Comment:
Do we also show the PID and the other minimal metadata for each annotation in the card?

Reply:
At the moment only the value, I can add the PID, would we want to add all the metadata like in the side panel?

Reply:
Yes that would be great! Maybe in a separate story so we can close this PR